wal/src/message_queue_impl/region.rs

```diff
@@ -697,12 +1007,10 @@ impl<C: ConsumeIterator> MessageQueueLogIterator<C> {
                 msg: "failed while polling log",
             })?;
 
-        self.previous_value = payload.to_owned();
-
         Ok(Some(LogEntry {
             table_id: log_key.table_id,
             sequence: log_key.sequence_num,
-            payload: self.previous_value.as_slice(),
+            payload: payload.to_owned(),
```

Comment:
Actually, this will lead to another copy. Is it necessary to eliminate the lifetime from the returned value?

Reply:
It is not necessary, just for adapting to the trait defined in `wal/src/manager.rs`.
It can actually avoid cloning in some implementation such as rocksDB, so I think keeping it is no problem.

Reply:
Because I need to wrap this iterator as a inner in later development, and if return slice here, the borrow checker may complain... The refactor here just for making it happy.

Reply:
Has reverted this refactor.